lib/std/ascii.zig

```diff
@@ -179,20 +179,12 @@ pub fn isASCII(c: u8) bool {
 
 /// Uppercases the character and returns it as-is if already uppercase or not a letter.
 pub fn toUpper(c: u8) u8 {
-    if (isLower(c)) {
```

Comment:
I like the documentation and pointer quality changes but I disagree with these kind of changes. I don't think it necessarily makes it easier to read, rather the opposite sometimes. I think it might be better if you follow the existing style and base your changes on that instead of trying to squeeze stuff on as little lines as possible.

Reply:
I have reverted those changes back to their original form.